test/pjrt/test_experimental_pjrt_gpu.py

```diff
@@ -11,14 +11,15 @@
 import torch_xla.core.xla_env_vars as xenv
```

Comment:
should we renamed these `experimental` files?

Reply:
Yes, good catch.